runtime/pendulum/src/xcm_config.rs

```diff
@@ -1,5 +1,5 @@
 use super::{
-	AccountId, Balance, Balances, CurrencyId, ParachainInfo, ParachainSystem, PolkadotXcm, Runtime,
```

Comment:
If we don't export the `CurrencyId` as `PendulumCurrencyId` as I suggest in the other comment, we can keep this.

Reply:
Done!